src/ape/cli/__init__.py

```diff
@@ -27,6 +27,7 @@
     verbosity_option,
 )
 from ape.cli.paramtype import AllFilePaths, Path
+from ape.plugins._utils import PIP_COMMAND
```

Comment:
I wasn't super clear I guess before! I meant remove this from this module entirely and import only from _utils everywhere. Why does this need to be defined here and in `__all__` ?

Not saying am against it, just mostly trying to follow. If we do decide to keep `PIP_COMMAND `in ape.cli` namespace, you can put it back, as that would make sense then.

Reply:
it looks like nothing is importing from here too, maybe it was leftover accidentally? does the circular go away if deleted?

Reply:
I have no idea why it's here

Reply:
removed in 125a616d89db1be3e0c2993078ef920d1829f6b2